helm/aws-load-balancer-controller/values.yaml

```diff
@@ -213,6 +213,9 @@ syncPeriod:
 # Namespace the controller watches for updates to Kubernetes objects, If empty, all namespaces are watched.
 watchNamespace:
 
+# disableIngressValidation disables the validation of resources of kind Ingress, false by default
```

Comment:
Lets group this configuration under webhookConfig, that way we can group the webhook related config together. Something like this:
```
# webhookConfig contains configurations specific to the validating and mutating webhooks
webhookConfig:
    # disableIngressValidation disables the validation of ingress resources, false by default
    disableIngressValidation: false
```

Reply:
Done. I placed the webhookConfig above the existing webhook* keys.